### ISSUE TITLE
READMEに載っていないconfigの設定項目の追記

### ISSUE THREAD

**Issue by tanaka-hiroki1989** [clinfo/DeepKF#25]
- label_train_npy : 学習用ラベル
- label_test_npy : テスト用ラベル
- label_type : ラベルのタイプ
- task : タスク（generative / label prediction）
- time_major
- epoch_interva_eval : 何epochごとにvalidation_costを評価するかを指定する
- curriculum_beta : パラメータbetaを学習中に変化させるかどうか
- curriculum_gamma : パラメータgammaを学習中に変化させるかどうか
- dropout_rate : ドロップアウト率
- learning_rate : 学習率
- potential_grad_delta : 
- filed_grid_num : 
- field_grid_dim:
